packages/stark-core/src/modules/http/constants/http-headers.constants.ts

```diff
@@ -18,44 +18,4 @@ export class StarkHttpHeaders {
 	 * The media type of the body request
 	 */
 	public static CONTENT_TYPE: string = "Content-Type";
-	/**
-	 * Used to prevent Cross Site Request Forgery
-	 */
-	public static XSRF_TOKEN: string = "X-XSRF-TOKEN";
```

Comment:
This header should not be removed since it is not NBB specific... or at least we should keep it for now until we tackle this issue #115